references/detection/transforms.py

```diff
@@ -437,3 +439,114 @@ def forward(
                 )
 
         return image, target
+
+
+def _copy_paste(image, target, paste_image, paste_target, inplace=True):
+
+    # Random paste targets selection:
+    num_masks = len(paste_target["masks"])
+    random_selection = torch.randint(0, num_masks, (num_masks,)).unique()
+
+    paste_masks = paste_target["masks"][random_selection]
+    paste_boxes = paste_target["boxes"][random_selection]
+    paste_labels = paste_target["labels"][random_selection]
+
+    masks = target["masks"]
+    # If source and paste data have different sizes
+    # Let's resize paste data
+    size1 = image.shape[-2:]
+    size2 = paste_image.shape[-2:]
+    if size1 != size2:
+        paste_image = F.resize(paste_image, size1, interpolation=F.InterpolationMode.BICUBIC)
```

Comment:
This interpolation should probably be provided as argument to align with the interpolation mode used elsewhere.

Reply:
OK, I can expose it as an argument. I put it hard-coded as resize and also gaussian_blur are just helping ops and not "essential" of the method.

Reply:
As seen offline, detection reference does not expose args for interpolation mode and has a bit particular way to resize image and mask inside the model. Let's keep it as it is for now.